echo-web/src/main/java/com/netflix/spinnaker/echo/scheduler/TriggerListResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.netflix.spinnaker.echo.scheduler;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+public class TriggerListResponse {
+  List<TriggerDescription> pipeline;
```

Comment:
Can these variables be `private` (and if possible `final`)?

Reply:
Yes! thanks!